docs/howto/how-to-make-api-tool/api-tool-guide.md

```diff
@@ -0,0 +1,250 @@
+            ftm.append(stat["ftm"])
+            tos.append(stat["turnover"])
+        
+        table = pa.table(
+            [points, rebounds, assists, steals, blocks, minutes, fga, fgm, fg3, fta, ftm, tos],
+            schema=stats_chema
+        )
+        return table
+```
+This function iterates over the returned JSON from step 1 and pulls the relevant fields into an array of arrays, which conforms to the [`pa.schema`](https://arrow.apache.org/docs/python/generated/pyarrow.Schema.html) that we defined in the beginning of the function.
+
+> Other times, the data comes in a better format and you can convert to Apache Arrow format more easily with the [built-in helper function](https://arrow.apache.org/docs/python/json.html) 
+### 3. Getting the min, max, and mean 
+Now that we have a `pyarrow.Table` representation of the JSON data, we can use the built-in [compute functions](https://arrow.apache.org/docs/python/compute.html) to calculate the min, max, and mean of each of our statistical categories with the following function:
+
+```python
+    def compute_aggregate_output(table):
+        mins = []
+        maxs = []
+        means = []
+        cat = []
+        for stat, col_name in zip(table, table.column_names):
+            if stat.type == pa.duration('s'):
+                stat = stat.cast(pa.int64())
+            min_max = pc.min_max(stat)
+            mins.append(min_max[0].as_py())
+            maxs.append(min_max[1].as_py())
+            means.append(pc.mean(stat).as_py())
+            cat.append(col_name)
+        
+        aggregate_tables = [cat, means, mins, maxs]
+
+        return pa.table(aggregate_tables, names = ["category", "mean", "min", "max", ])
+```
+
+This function will call the compute function `pc.min_max()` and `pc.mean()` on each statistical category of our `pa.Table` that we created in step 2. It then aggregates all of these into a new table and returns it.
+
+### 4. Conclusion
+Finally, we'll combine everything we did in steps 1-3 into the `on_complete` function and write the results to the output anchor. It should look like this:
+
+```python
+    def on_complete(self) -> None:
+        """
+        In this method, a Plugin designer should perform any cleanup for their plugin.
+        However, if the plugin is an input-type tool (it has no incoming connections),
+        processing (record generation) should occur here.
+        """
+        import pyarrow as pa
+        import pyarrow.compute as pc
+
+        LEBRON_JAMES_PLAYER_ID = 237
+        NBA_SEASON = 2016
+        lbj_stats = get_postseason_stats(LEBRON_JAMES_PLAYER_ID, NBA_SEASON)
+        table = get_pyarrow_table_from_stats(lbj_stats)
+        output_table = compute_aggregate_output(table)
+        self.provider.write_to_anchor("Output", output_table)
+        self.provider.io.info("APITool tool done.")
+```
+
+And now you're done with writing the code! Building and running this tool in Designer should yield the following output:
```

Comment:
TW review - update to:

And now you're done with the code! When you build and run this tool in Alteryx Designer, it should yield the this output:

Reply:
hey david, could you clarify what i should update this to? 

the `it should yield the this output` looks like a typo

Reply:
@ayx-orion good catch sorry! it should say:

And now you're done with the code! When you build and run this tool in Alteryx Designer, it should yield this output: